lnbits/templates/components.vue

```diff
@@ -631,6 +632,9 @@
       class="qrcode__image"
       :src="custom.logo"
       alt="qrcode icon"
+      @click="downloadSVG"
```

Comment:
some QR codes are wrapped in an <a> tag!

won't this cause issues?

Reply:
now there if `:href=""` where you can pass a url